### ISSUE TITLE
Cannot disable an ion-list button on Ionic 2

### ISSUE THREAD

**Issue by boaz-chen** [driftyco/ionic#6108]
I have the following view which doesn't work as expected (button isn't disabled):

`
<ion-list>
       <button ion-item disabled>
           My Button
       </button>
   </ion-list>`

Ionic Version: 2.0.0-beta.3

**Comment by johnfakeacc** [comment]
In my opinion: You cannot combine button with ion-item directive?!

**Comment by danielcrk-cn** [comment]
You can:
http://ionicframework.com/docs/v2/api/components/item/Item/

"Use <button ion-item> for something that can be clicked/tapped. Typically this element will also have a (click) handler."

**Comment by boaz-chen** [comment]
As a workaround for now I've defined the following CSS:

`
.item-disabled, item-disabled * {
  opacity: 0.3;
  pointer-events: none; 
  }
`

And then in my HTML:

`<ion-list> <button ion-item class="item-disabled"> My Button </button> </ion-list>`

**Comment by brandyscarney** [comment]
This has been added to the `beta.8` release. If you'd like to check it out sooner, I've released a nightly version that can be installed by running the following in a terminal:

```
npm install --save ionic-angular@2.0.0-beta.7-201605202104
```

If you haven't updated to beta.7 yet, check out the [changelog](https://github.com/driftyco/ionic/blob/2.0/CHANGELOG.md#angular-update-to-200-rc1) for steps to update.

Thanks!
